### PR TITLE
Update LICENSE & NOTICE

### PR DESCRIPTION
**Pull request by oschaaf**
Some changes as per discussed on the email thread. 
Initiating this PR for addressing issues that need to be fixed before we can attempt a release.

Signed-off-by: Otto van der Schaaf <oschaaf@we-amp.com>

### CODE REVIEW

**Review by jmarantz — APPROVED** [review]

**Review by jmarantz — APPROVED** [review]

File: LICENSE [hunk 1]
```diff
@@ -176,17 +176,7 @@
 
    END OF TERMS AND CONDITIONS
 
-   APPENDIX: How to apply the Apache License to your work.
```
Comment (jukka) [inline LICENSE]: Please keep this part.

File: NOTICE [hunk 2]
```diff
@@ -1,110 +1,94 @@
...
 
-Files: src/third_party/google-sparsehash/*
-Copyright: 2005 Google Inc.
-License: BSD-3-clause
+Notice utf
 
-Files: src/third_party/libwebp/*
-Copyright: 2010 Google Inc.
-License: BSD-3-clause
+/*
```
Comment (jukka) [inline NOTICE]: It's sufficient to keep this part just in the LICENSE file.

File: LICENSE (hunk 1 above)
Comment (oschaaf) [reply LICENSE]: Whoops, done

File: NOTICE (hunk 2 above)
Comment (oschaaf) [reply NOTICE]: removed this here.

**Review by oschaaf — COMMENTED** [review]
@jukka feedback addressed in dd6e061bb986094a9189f100e384540122f8639b
The changes to the script that produce the tarball may need a follow up if there are dependent
scripts. I also need to test its result, to double check its contents are buildable / pass tests.
These things would bloat this PR, so I'll do that later.

**Comment by oschaaf** [comment]
Pushed a small change to fix up `devel/build_release_tarball.sh` for the adjusted paths.

**Pull request merged**